python/sanlock.c

```diff
@@ -184,6 +187,17 @@ add_align_flag(long align, uint32_t *flags)
     return 0;
 }
 
+static void
+util_set_value_error(const char* format, PyObject* obj)
```

Comment:
Why we need the util_ prefix?

set_value_error() is a static function and It should not conflict with any other api we use.